CHANGELOG.md

```diff
@@ -1,3 +1,58 @@
+## 0.46.0
+
+### ⚠️ Breaking changes
+
+* Align implicit type casting behavior of `match` expressions with with `case/==` ([#6684](https://github.com/mapbox/mapbox-gl-js/pull/6684))
+
+### ✨ Features and improvements
+
+* :tada: Add `Map#setFeatureState` and `feature-state` expression to support interactive styling ([#6263](https://github.com/mapbox/mapbox-gl-js/pull/6263))
+* Create draggable `Marker` with `setDraggable` ([#6687](https://github.com/mapbox/mapbox-gl-js/pull/6687))
+* Add `Map#listImages` for listing all currently active sprites/images ([#6381](https://github.com/mapbox/mapbox-gl-js/issues/6381))
+* Add "crossSourceCollisions" option to disable cross-source collision detection ([#6566](https://github.com/mapbox/mapbox-gl-js/pull/6566))
+* Handle `text/icon-rotate` for symbols with `symbol-placement: point` ([#6075](https://github.com/mapbox/mapbox-gl-js/issues/6075))
+* [andrewharvey](https://github.com/andrewharvey) cleans up:
```

Comment:
Oh yeah I was using that at first, but the `changelog-draft` tool used the link format so I switched to that just to be consistent.

Reply:
I appreciate it, but you don't need to mention me. If you insist though I'd prefer it as (h/t) at the end of each line like the others, as it makes scanning though the changelog easier and avoids grouping unrelated changes.

Reply:
@andrewharvey No problem, I'll switch to (h/t) in the changelog for the final release. And unless you ask us to remove your name from the changelog entirely, we'll "insist" on including it because we only have a few ways of showing our appreciation for your contributions and we want to make use of them. 😄